db/db_impl/db_impl.cc

```diff
@@ -3185,22 +3185,55 @@ void DBImpl::MultiGetEntity(const ReadOptions& _read_options, size_t num_keys,
                             ColumnFamilyHandle** column_families,
                             const Slice* keys, PinnableWideColumns* results,
                             Status* statuses, bool sorted_input) {
+  assert(statuses);
+
```

Comment:
QQ. Curious if we need to do anything when `num_keys` is 0.

Reply:
Don't think so - in general, we don't consider that an error and there's an early return for that case already in `MultiGetCommon`